### ISSUE TITLE
Repository: Wiki synchronization not working

### ISSUE THREAD

**Issue by Vankata453** [SuperTux/supertux#2206]
Currently, the "Wiki" tab of this repository isn't being synchronized to the SuperTux [wiki](https://github.com/SuperTux/wiki) repository. Even if there are new changes to the "wiki" repository, they are not being applied here, which removes the ability of allowing users, who do not have access to edit the "Wiki" in this repository, to edit it, using pull requests. Synchronization not working may cause other inconveniences as well.

**Comment by Mathnerd314** [comment]
I fixed the wiki tab -> wiki repo sync direction, it hadn't been used for a long time so the SSH key got deleted. That direction has a smart script that I wrote, so now that it's run the two repos are synced.

For the future though it also seems necessary to change the wiki repo -> wiki tab direction to make it 'smart', based on the error messages. The only issue is if the two hooks trigger each other, then there could be an infinite loop.
